src/renderer/shared/lib/utils/constants.ts

```diff
@@ -58,3 +58,6 @@ export const DEFAULT_TIME = new BN(6_000);
 export const ONE_DAY = new BN(24 * 60 * 60 * 1000);
 
 export const MONTH = 30 * 24 * 60 * 60 * 1000;
+
+export const CHAINS_CONFIG_URL = `https://raw.githubusercontent.com/novasamatech/nova-spektr-utils/main/chains/${process.env.CHAINS_VERSION}/${process.env.CHAINS_FILE + '.json'}`;
```

Comment:
IMO chains version is knowledge related to integration logic, it should be configured from inside of the app, not outside